docs/release_notes/0.5.1.md

```diff
@@ -0,0 +1,10 @@
+# Release 0.5.1
+
+## Features
+
+- Add support for Kubernetes 1.14 (#1281)
```

Comment:
I thought we said that items begin with lower-case, which is what I used in 0.5.0 notes...

Reply:
The 0.5.0 notes are not in master. I looked at the 0.5.0-rc.0.md and thought we've started using capitalised words to start items.

Reply:
I updated the notes to use lowercase letters.